host_vars/au1/vars.yaml

```diff
@@ -1,4 +1,4 @@
-spads_hostRegion: '"AU - "'
-spads_lobbyLogin: '[teh]clusterAU'
+spads_hostRegion: '"AU1 - "'
```

Comment:
Now that all will be consistent, instead of changing every single host file, you can template variables in https://github.com/beyond-all-reason/ansible-spads-setup/blob/main/group_vars/prod/vars.yaml like I've done it for local dev: https://github.com/beyond-all-reason/ansible-spads-setup/blob/8e2a285f979f72898f7c2e470716b1fa2754b573/group_vars/dev.yaml#L2-L7

This way no repetition is needed in any of the files.

Reply:
Like so?
```
spads_hostRegion: '"{{inventory_hostname | upper}} - "'
spads_lobbyLogin: 'Host[{{inventory_hostname | upper}}]'
spads_channel: cluster{{inventory_hostname | upper}}
spads_maxInstances: 80
```

Reply:
Yes im renaming existing ones